src/components/Icon.tsx

```diff
@@ -0,0 +1,27 @@
+import { FunctionComponent, SyntheticEvent } from 'react'
+
+import * as icons from '@mui/icons-material'
+
+interface Icon {
+    name: keyof typeof icons
+    size: number
+    className?: string | undefined
+    color?: string
+    key?: string
+    onClick?: (event: SyntheticEvent) => void
+    variant?: 'boxed'
+}
+
+export const Icon: FunctionComponent<Icon> = ({ name, size, className = '', color, key, onClick, variant }) => {
+    const IconName = icons[name]
+
+    return (
+        <IconName
+            className={`${className} ${variant === 'boxed' ? 'bg-violet-mist text-vivid-violet rounded p-2' : ''}`}
+            style={{ fontSize: size }}
```

Comment:
I wonder if we can put in a default size too so it won't always be needed for the majority of icons put in the body. Seems like 48 or 21? Was that talked about with Tracey? I think it'd help reinforce our standardizations.

Reply:
This is an excellent point, considering the potential for variations! Tracey's Figma specifies the new icon style with a size of 48, so let's go with that as the default. We can flag this as a decision point for her in stakeholder review. 

In 9af1fa6, I added this as the default size and removed instances where `size={48}` is used on the component. 

You have me thinking. Ultimately, we might emulate what we're doing here with `variant` so that `size` can only be chosen from categories determined in this component. For example, the `Icon` component would accept a string corresponding with the uniform sizes of nav arrows, iconography, or social media icons. Because that adds some maintenance, let's first checkout what Tracey has planned. Great to know what you think of this idea though.

Reply:
I like that idea! Takes it a step further & can help us standardization body icons, command icons (for arrows), etc.